compiler/jsgen.nim

```diff
@@ -1434,8 +1434,19 @@ proc genAddr(p: PProc, n: PNode, r: var TCompRes) =
     of nkConv:
       genAddr(p, n[0], r)
     of nkStmtListExpr:
-      if n.len == 1: gen(p, n[0], r)
```

Comment:
This check makes no sense, of course `n.len == 1` else it would not be an `addr` expression.